src/testers.jl

```diff
@@ -191,6 +192,9 @@ function test_rrule(
         ∂self = ∂s[1]
         x̄s_ad = ∂s[2:end]
         @test ∂self === NoTangent()  # No internal fields
+        msg = "The pullback must return `(∂self, ∂args...)`, where `length(∂args) == " *
+            "length(args)`, and `args` are the arguments of the primal function."
+        @test_msg msg length(x̄s_ad) == length(inputs)
```

Comment:
Maybe we should rename these to be nicer in failure message? Or maybe they are good enough